### DISCUSSION TITLE
ListCompareAlgorithm.AS_SET is not working after changing element values.

### DISCUSSION THREAD

**Discussion by tsidheeque** [javers/javers#1348] [General]
When I am using ListCompareAlgorithm.AS_SET, ignoring the orders but any element value difference its giving lot of differences. 

Left_Side - 
"country_code" : "AB",
"country_name" : "ABCD",
**"zip_code": "12345"**
"phone_number" : "9999"

Right_Side
LeftSide - 
"country_code" : "AB",
"country_name" : "ABCD",
**"zip_code": "12346"**
"phone_number" : "9999"

Based on the above example zip_code value changed, but after comparing getting 4 difference in leftside (considering everything as null). could you please provide the solution for the same 

  - 'value' collection changes :
removed
added
removed
added
unset

Thank you

**Comment by bartoszwalacik** [comment]
Hi @tsidheeque , please clarify this case. Start from providing a runnable test case, see [guidelines-for-bug-reporting](https://github.com/javers/javers/blob/master/CONTRIBUTING.md#guidelines-for-bug-reporting)
